cloudinit/sources/azure/errors.py

```diff
@@ -48,10 +50,9 @@ def as_description(
             f"timestamp={self.timestamp.isoformat()}",
         ]
 
-        system_uuid = dmi.read_dmi_data("system-uuid")
-        if system_uuid:
-            system_uuid = system_uuid.lower()
-            error.append(f"vm_id={system_uuid}")
+        vm_id = query_vm_id()
```

Comment:
it will be useful to report this via diagnostic KVP, even for the successful case

Reply:
I will add a diag in a separate PR.